src/main/java/com/google/sps/servlets/test.java

```diff
@@ -0,0 +1,58 @@
+package com.google.sps.servlets;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.json.simple.JSONObject;
+
+@WebServlet("/test")
+public class test extends HttpServlet
+{
+  
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException
+  {
+       Connection conn; // Cloud SQL connection 
+      try
+      {
+          conn = DriverManager.getConnection("jdbc:mysql:///team5project?cloudSqlInstance=wramos-sps-summer21:us-central1:team5project&socketFactory=com.google.cloud.sql.mysql.SocketFactory&user=root&password=spsTeam5&useSSL=false"); 
+          ResultSet rs = conn.prepareStatement("SELECT * FROM QUESTIONS").executeQuery(); 
```

Comment:
Your SQL code would be more maintainable going forward if you always enumerate the columns that you want and add a `limit` clause.